### ISSUE TITLE
statusbox crash when flipping through modes

### ISSUE THREAD

**Issue by jamessanford** [jamessanford/assetto-corsa-deltabar#4]
I was clicking through the various modes (best sector, optimal, ...) repeatedly and it crashed.

Looks like statusbox/format_time() expects milliseconds to only ever be an integer...  There are likely more bugs in statusbox.

```
Exception: Thu Sep 24 05:24:46 2015
Traceback (most recent call last):
  File "apps/python/deltabar\deltabar.py", line 59, in acUpdate
    deltabar_app.acUpdate(delta_t)
  File "apps/python/deltabar\deltabar_lib\deltabar_lib.py", line 356, in acUpdate
    pos, self.lap.invalid_sectors[current_sector])
  File "apps/python/deltabar\deltabar_lib\statusbox.py", line 221, in update_frame
    self.data.textbox.set('i{}_2'.format(current_sector), format_time(current_sector_time - benchmark_sector_time, invalid=invalid, delta=True))
  File "apps/python/deltabar\deltabar_lib\statusbox.py", line 71, in format_time
    return "{}{}{}.{:03d}".format(star, sign, sec, thou)
ValueError: Unknown format code 'd' for object of type 'float'
```

**Comment by StoneSpb** [comment]
Hi, James!

Thanks, will fix it :)

 James Sanford  notifications@github.com:

**Comment by jamessanford** [comment]
Oh, I just fixed it :)

**Comment by StoneSpb** [comment]
Mmm!
Thank you! :)

Alexander Denisov

James Sanford  notifications@github.com:

**Comment by jamessanford** [comment]
"fixed" in 469c7e9f352afd1b745dbd349b1dec315dbe1a56
